tsconfig.json

```diff
@@ -7,7 +7,7 @@
 		"declaration": true,
 		"pretty": true,
 		"removeComments": true,
+		"skipLibCheck": true,
```

Comment:
Needed because `tsc` was failing in type checks for the `strict-event-emitter-types` package. The `skipLibCheck` flag is used in many other balena packages.

Reply:
Yep, should definitely be included.